Modules/src/UserInputValidator.cc

```diff
@@ -72,9 +72,17 @@ namespace ChimeraTK {
     }
 
     for(auto& accessor : _module->getAccessorListRecursive()) {
-      if(accessor.getDirection() == VariableDirection{VariableDirection::feeding, true} &&
-          accessor.getModel().getTags().count(std::string(UserInputValidator::tagValidatedVariable)) > 0) {
-        _downstreamInvalidatingReturnChannels.emplace(accessor.getAppAccessorNoType().getId());
+      std::cout << "Evaluating accessor " << _module->getName() + accessor.getName() << std::endl;
```

Comment:
This looks like forgotten debug output!

Reply:
Argh, that file should have been stashed.

Reply:
reverted it to the version in the master